app/components/MazemapEmbed/index.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+import { useEffect } from 'react';
+import "node_modules/mazemap/mazemap.min.css"
+import * as Mazemap from 'mazemap';
+
+type Props = {
+  sharepoi: number,
+  height?: number,
+};
+
+/** A component that shows a mazemap map of a given poi (e.g. room),
+ * largely based on https://api.mazemap.com/js/v2.0.12/docs/#ex-data-poi
+ */
+
+export const MazemapEmbed = ({ sharepoi, ...props }: Props) => {
+  //useEffect to initialize only once, sharepoi will probably not change
+  useEffect(() => {
+    // $FlowFixMe Mazemap is defined in script tag, has no types
+    const myMap = new Mazemap.Map({
+      container: 'mazemap-embed',
+      campuses: 1,
+      center: { lng: 10.4042965, lat: 63.4154135 },
+      zLevel: 3,
+      zoom: 16,
+      minZoom: 10,
+      maxZoom: 20,
+      zLevelControl: true,
+      scrollZoom: true,
+      doubleClickZoom: false,
+      touchZoomRotate: false,
+    });
+    myMap.on('load', () => {
+      // Initialize a Highlighter for POIs
+      // Storing the object on the map just makes it easy to access for other things
+      myMap.highlighter = new Mazemap.Highlighter(myMap, {
+        showOutline: true, // optional
+        showFill: true, // optional
+        outlineColor: Mazemap.Util.Colors.MazeColors.MazeBlue, // optional
+        fillColor: Mazemap.Util.Colors.MazeColors.MazeBlue, // optional
+      });
```

Comment:
Maybe we don't need the `// optional` docs from `mazemap` here?

Reply:
Yes, I guess they're _optional_.